cmd/server/cadence/server.go

```diff
@@ -108,9 +108,10 @@ func (s *server) startService() common.Daemon {
 	params := service.BootstrapParams{}
 	params.Name = "cadence-" + s.name
 	params.Logger = loggerimpl.NewLogger(s.cfg.Log.NewZapLogger())
+	params.UpdateLoggerWithServiceName(s.name)
```

Comment:
In general this makes a lot of sense.

However this is not the only place where `service.BootstrapParams` parameters are constructed. There are:
- `host/onebox.go` used for integration tests. It needs updating as well.
- We also construct these params on the internal "Uber" version of Cadence. And we need to make sure to not forget adding this tag on the next release.

Reply:
Thanks for your review, I have also added the `UpdateLoggerWithServiceName` to other places in onebox.